asdf/tests/helpers.py

```diff
@@ -113,8 +118,8 @@ def recurse(old, new):
 
 
 def assert_roundtrip_tree(
-        tree, tmpdir, asdf_check_func=None, raw_yaml_check_func=None,
-        write_options={}, extensions=None):
+        tree, tmpdir, asdf_check_func=None, tree_match_func='assert_equal',
```

Comment:
I feel that `asdf_check_func` and `tree_match_func` are a little redundant here. Do you think it makes sense to remove `asdf_check_func`?

Reply:
It does seem redundant. We'll have to go through and update all of the test cases that currently make use of it.